db/hybrid_logical_vector.go

```diff
@@ -443,6 +414,36 @@ func (hlv *HybridLogicalVector) toHistoryForHLV() string {
 	return s.String()
 }
 
+// ToDecodedHybridLogicalVector converts the little endian hex values of a HLV to uint64 values
+func (hlv *HybridLogicalVector) ToDecodedHybridLogicalVector() DecodedHybridLogicalVector {
+	var decodedVersion, decodedCVCAS, decodedImportCAS uint64
+	if hlv.Version != "" {
+		decodedVersion = base.HexCasToUint64(hlv.Version)
+	}
+	if hlv.ImportCAS != "" {
+		decodedImportCAS = base.HexCasToUint64(hlv.ImportCAS)
+	}
+	if hlv.CurrentVersionCAS != "" {
+		decodedCVCAS = base.HexCasToUint64(hlv.CurrentVersionCAS)
+	}
+	decodedHLV := DecodedHybridLogicalVector{
+		CurrentVersionCAS: decodedCVCAS,
+		Version:           decodedVersion,
+		ImportCAS:         decodedImportCAS,
+		SourceID:          hlv.SourceID,
+		PreviousVersions:  make(map[string]uint64),
+		MergeVersions:     make(map[string]uint64),
```

Comment:
Could pre-allocate size for these maps

```suggestion
		PreviousVersions:  make(map[string]uint64, len(hlv.PreviousVersions)),
		MergeVersions:     make(map[string]uint64, len(hlv.MergeVersions)),
```